packages/relay/tests/lib/subscriptionController.spec.ts

```diff
@@ -29,19 +29,24 @@ import sinon from 'sinon';
 import dotenv from "dotenv";
 import path from 'path';
 import {Registry} from "prom-client";
+import ConnectionLimiter from "@hashgraph/json-rpc-ws-server/dist/ConnectionLimiter";
```

Comment:
Q: why is this file in relay vs in `ws-server` package again?
Shouldn't we move it to `ws-server`?

Reply:
I was thinking that we want to keep the business logic in the `relay` package, and the `server` and `ws-server` are more related to routing. It does make sence to move it to `ws-server`, we can do that when we start with the refactoring of the project.

Reply:
@Ivo-Yankov  Okay please open a ticket so we don't miss it and we can plan it